behat_ibexa_experience.yaml

```diff
@@ -39,6 +39,7 @@ regression:
               - '%paths.base%/vendor/ibexa/user/features/browser'
               - '%paths.base%/vendor/ibexa/workflow/features/browser'
               - '%paths.base%/vendor/ibexa/segmentation/features'
+              - '%paths.base%/vendor/ibexa/version-comparison//features'
```

Comment:
Additional '/' in path.

Reply:
Good catch, thanks! Fixed in https://github.com/ibexa/behat/pull/39/commits/03b70c22efd9b0deb61e67ee791acdb50798d400